kwanghwi/myapp/app.js

```diff
@@ -37,23 +37,135 @@ app.get("/ping", (req, res) => {
 });
 
 app.post("/user/signup", async (req, res) => {
-  const { userId, password, email, profileImage } = req.body;
+  const { userId, password, name, email, profileImage } = req.body;
   await appDataSource.query(
     `INSERT INTO users(
-      user_id,
+      userId,
       password,
+      name,
       email,
       profileImage
-    ) VALUES (?, ?, ?, ?);
+    ) VALUES (?, ?, ?, ?, ?);
     `,
-    [userId, password, email, profileImage]
+    [userId, password, name, email, profileImage]
   );
   return res.status(201).json({ message: "signup success!" });
 });
 
+app.post("/post/create", async (req, res) => {
+  const { title, content, content_image, userId } = req.body;
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      content_image,
+      user_id
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, content_image, userId]
+  );
+  return res.status(201).json({ message: "postcreate success!" });
+});
+
+app.get("/posts/get", async (req, res) => {
+  await appDataSource.query(
+    `SELECT
+            u.id as user_id,
+            u.profile_image as userProfileImage,
+            p.id as postingId,
+            p.content_image as postingImageUrl,
+            p.content as postingContent
+        FROM posts p
+    INNER JOIN users u ON u.id = p.user_id;
+    `,
+    (err, rows) => {
+      return res.status(200).json({ data: rows });
+    }
+  );
+});
+
+app.get("/post/get/:userId", async (req, res) => {
```

Comment:
```suggestion
app.get("/users/:userId/posts", async (req, res) => {
```
해당 endpoint는 유저들 중, 특정 유저의, 게시글 목록을 불러오는 것 같네요! 맞다면 다음과 같이 수정되면 좋을 것 같습니다.